helix-core/src/test/java/org/apache/helix/model/TestClusterConfig.java

```diff
@@ -393,4 +398,32 @@ private void trySetInvalidAbnormalStatesResolverMap(ClusterConfig testConfig,
       // expected
     }
   }
+
+  @Test
+  public void testSuperClusterRebalanceConfig() {
+    ClusterConfig testConfig = new ClusterConfig("testConfig");
+
+    // Check the default values
+    Assert.assertEquals(testConfig.getSuperClusterRebalanceConfig().getRebalanceStrategy(),
+        CrushEdRebalanceStrategy.class.getName());
+    Assert.assertEquals(testConfig.getSuperClusterRebalanceConfig().getRebalanceClassName(),
+        DelayedAutoRebalancer.class.getName());
+    Assert.assertEquals(testConfig.getSuperClusterRebalanceConfig().getRebalanceMode(),
+        RebalanceConfig.RebalanceMode.FULL_AUTO);
+
+    // Check set and read
+    RebalanceConfig rebalanceConfig = testConfig.getSuperClusterRebalanceConfig();
+    rebalanceConfig.setRebalanceClassName(WagedRebalancer.class.getName());
```

Comment:
Do we plan to also support REST API for this change? Feel that'll be very useful.

Reply:
Rest API already supports modify cluster config. There is no need to add a specific method just for supercluster configuration, IMO. It is debatable for sure. But either way, we can do this PR first.

Reply:
I think REST API through cluster config is good enough. Maybe you can add one test case in `TestClusterAccessor`, similar as `testPurgeOfflineParticipants` so that your PR is complete.

Reply:
I doubt that it will increase any meaningful test coverage. 1. the REST API support is not the focus of this change. 2. There is no change in the REST, I think the existing tests cover the logic well.

To clarify, my point is that I don't have a good case that will pass the existing HelixAdmin tests but fail a REST test (because of a relevant change in this PR). So I think another REST test is just adding a boring layer to the already covered logic.
But if you do have such a case, please let me know and I will be glad to add the test.